src/Controller/SponsorshipBenefit/Edit.php

```diff
@@ -37,20 +38,15 @@ public function __construct(
 
     /**
      * @param Request $request
-     * @param string $id
+     * @param SponsorshipBenefit $sponsorshipBenefit
```

Comment:
As above, would it be better not to use the paramconverter and retrieve it from the `$request`?